src/main/java/org/roaringbitmap/RunContainer.java

```diff
@@ -1506,6 +1506,10 @@ protected boolean isFull() {
     return (this.nbrruns == 1) && (this.getValue(0) == 0) && (this.getLength(0) == -1);
   }
 
+  public static Container full() {
+    return new RunContainer(1, new short[]{0, -1, 0, 0, 0, 0, 0, 0});
```

Comment:
4 is just an arbitrary value. As for the method ```iadd```, I think the implementation can be optimized.